pkg/tag/tag.go

```diff
@@ -47,13 +49,34 @@ func (il SortableImageTagList) Swap(i, j int) {
 
 // NewImageTag initializes an ImageTag object and returns it
 func NewImageTag(tagName string, tagDate time.Time, tagDigest string) *ImageTag {
+	return NewImageTagWithTagPart(tagName, tagName, tagDate, tagDigest)
+}
+
+// NewImageTagWithTagPart initializes an ImageTag object and returns it
```

Comment:
Please be more specific what the difference to `NewImageTag` is. Also I was wondering if this should be rather something more along the lines of the following, instead of a new initializer:

```go
func (t *ImageTag) WithTagNamePart(tagNamePart string) *ImageTag {
  t.TagNamePart = tagNamePart
  return t
}

tag := NewImageTag(...).WithTagNamePart(...)

```
WDYT?